sdk/storage/storage-blob/README.md

```diff
@@ -179,8 +151,37 @@ async function main() {
     `Create container ${containerName} successfully`,
     createContainerResponse.requestId
   );
+```
+
+### List the containers
```

Comment:
Can also add a link to the iterators sample ?

Reply:
adding

---
In reply to: [295982769](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r295982769) [](ancestors = 295982769)